src/server/pps/server/api_server.go

```diff
@@ -996,7 +1034,7 @@ func (a *apiServer) pipelineManager(ctx context.Context, pipelineInfo *pps.Pipel
 		}
 
 		// Start worker pool
-		a.workerPool(ctx, PipelineRcName(pipelineInfo.Pipeline.Name, pipelineInfo.Version))
+		a.workerPool(ctx, PipelineRcName(pipelineInfo.Pipeline.Name, pipelineInfo.Version), RetriesPerDatum)
```

Comment:
It seems strange that we pass this constant into every invocation of `workerPool`.  Could `newWorkerPool` simply use this constant itself?